examples/SDL2/main.cpp

```diff
@@ -0,0 +1,15 @@
+#include<iostream>
+#include<SDL/SDL.h>
```

Comment:
> If it helps I checked what its trying to download...

it's okay, it means "cache miss" (error `22`) => Hunter need to buid package from sources. Error `1` mean CMake internal error.